projects/aca-shared/src/lib/components/document-base-page/document-base-page.component.ts

```diff
@@ -90,7 +90,7 @@ export abstract class PageComponent implements OnInit, OnDestroy, OnChanges {
       });
 
     this.sharedPreviewUrl$ = this.store.select(getSharedUrl);
-    this.infoDrawerOpened$ = this.store.select(isInfoDrawerOpened).pipe(map((infoDrawerState) => !this.isOutletPreviewUrl() && infoDrawerState));
```

Comment:
these methods you are no longer using, are they used somewhere else or can be cleaned up?

Reply:
It's used in 2 other places in this component